### ISSUE TITLE
Implement registration question for self-report form

### ISSUE THREAD

**Issue by Murkantilism** [CitizenHealth/COVIDx#13]
# Description
- Allow users to register, associating their self-reported data with a non-anonymous account
- "Points" aka the gamification system may not exist at this point, omit the 

# Acceptance Criteria
- [ ] Add thank you a message:
>   Done!
>   Thanks for being a citizen scientist!
>   Your submitted data will always remain anonymous.
>   Register below.
- [ ] Implement four fully-functioning registration buttons:
  - Register with Facebook (presumably via OAuth 2.0?)
  - Register with Google (presumably via OAuth 2.0?)
  - Register with Email Address
  - Continue without registering

**Comment by BrennenHodge** [comment]
We're using [Firebase Authentication](https://firebase.google.com/docs/auth/?authuser=0) and already have Facebook & Google setup. 

 Here's a good example of a [Firebase UI](https://fir-ui-demo-84a6c.firebaseapp.com/#recaptcha=normal&emailSignInMethod=password) we can use, and also how the different registration processes work.

**Comment by afixoftrix** [comment]
Hi. I'm trying to get started on contributing to the project. I don't see an assignee on this issue. It looks like the current version of the app already has facebook and google buttons for registration. I am will also need access to the Figma if that is available --necessary for the location of the thank you message.

**Comment by BrennenHodge** [comment]
Hi @afixoftrix! DM me your email at https://chat.citizenhealth.io/ so I can add you to the Figma team account. 

You are correct -- we have authentication setup for Google & Facebook, but we need to figure out the Firebase "[Anonymous](https://firebase.google.com/docs/auth/web/anonymous-auth)" login.

No one has taken this issue, so it's yours if you want it.

**Comment by afixoftrix** [comment]
Yes, I will take this.

**Comment by afixoftrix** [comment]
For anonymous log in, the admin of the firebase console will have to login to the console and allow anonymous sign-in in the sign-in method tab.

**Comment by Murkantilism** [comment]
@BrennenHodge would that be you? Or @ohitsdoh maybe?

**Comment by BrennenHodge** [comment]
@Murkantilism & @afixoftrix Anonymous sign-in is already enabled.

**Comment by BrennenHodge** [comment]
@afixoftrix I'm not sure if you saw the update, but we're having a Zoom team meeting tomorrow (5/14) at 4 pm CT: https://us02web.zoom.us/meeting/register/tZEtdeugrD8iHNAF5bl46DT2O8sSYE1H3S46

It will be great if you can make it!

**Comment by afixoftrix** [comment]
@BrennenHodge Yes I saw catch up video (thanks for that) and the card on trello. I'll attend the meeting.

**Comment by Murkantilism** [comment]
Updated description to reflect Tripetto adoption
